src/mountscript.sh

```diff
@@ -247,12 +299,6 @@ search_free_local_ip_with_prefix()
                 continue
             fi
 
-            if egrep " \<$local_ip\> " "$MOUNTMAP" >/dev/null; then
-                # Avoid excessive logs. 
-                # vecho "$local_ip is in use by aznfs!"
-                continue
-            fi
-
             is_present_in_iptables=$(iptables-save -t nat | grep "^${local_ip}$" | wc -l)
```

Comment:
nyc, we can do this "iptables-save -t nat" once at the beginning and save that in a variable and then use that here.